<launch>
    <param name="_speed" type="double" value="2.0"/>
    <param name="_turn" type="double" value="1.0"/>
    <rosparam file="$(find robot_teleop)/config/skid_steer_controller.yaml" />
    <node pkg="teleop_twist_keyboard" type="teleop_twist_keyboard.py" name="joy_to_twist"/>
    <node pkg="robot_teleop" type="skid_steer_controller.py" name="skid_steer_controller"/>
</launch>